### DISCUSSION TITLE
How does AGH handle static.adguard.com? How to control it?

### DISCUSSION THREAD

**Discussion by EntropySmoke** [AdguardTeam/AdGuardHome#3855] [Q&A]
AdGuard Home creates and maintains connections to static.adguard.com and filter domains for updates.

Those connections do not show up in AGH interface. AGH GUI doesn't show how such connections are router? Do they use DNS servers specified in AGH? Do they encryption specified in AGH? There is no info on that in AGH GUI...

Is it possible to manually control connections to/from static.adguard.com ? I only need to check for all updates 1 time in 24 hours.

**Comment by ainar-g** [comment]
If I recall correctly, the system's resolver is used for those requests.  What would you propose we do instead?

**Comment by EntropySmoke** [comment]
I think using system resolver is how it should bet, but it would be nice for AGH to list queries to static.adguard.com, block list updates, all other domains allow users to block those requests. 

If AGH is to be a system resolver then it should provide information on all domains it contacts. Otherwise people have to resort to TCPDump and other 3rd party tools. If it wasn't for TCPDump, I wouldn't even know that AGH connects to domains not listed in AGH query log...

**Reply by ainar-g** [reply]
@ameshkov, what do you think?  We had a similar discussion about the optimistic cache recently.

**Reply by ameshkov** [reply]
It was like that in the older versions, but in the end, it proved to be rather confusing.

All internal requests can be found in AGH log though.
